test/banach/retry_test.clj

```diff
@@ -5,6 +5,40 @@
             [clojure.math.numeric-tower :as math]
             [banach.retry :as retry]))
 
+(def ctx-1 {:failures [(Exception. "Slow your roll")]})
+(def ctx-2 (update ctx-1 :failures conj (Exception. "KABOOM")))
+
+(deftest routing-tests
+  (let [strat (retry/routing
+               (comp odd? count :failures) identity
+               (constantly true) retry/give-up)]
```

Comment:
Do you think specialcasing `:else` would make it worse or better? (`:else` is the preferred fallback clause for cond; although keep in mind that's different, because in cond it works, accidentally, by virtue of being truthy.)